reactive/documentation/getting-started.adoc

```diff
@@ -0,0 +1,217 @@
+= Getting started with Hibernate Reactive
```

Comment:
FWIW I'd recommend removing this page entirely. You already have a getting started guide in the reference documentation, and we can totally put a link to that section of the documentation wherever we link to the getting started guide. Actually that's already what we do for other projects.

And it will spare you the boring task of updating this page every time you update the reference documentation...

Reply:
Done

We will probably need to tweak the doc a bit and create better getting started section but it should do for now